frontend/src/scripts/locale.js

```diff
@@ -1,55 +0,0 @@
-/*eslint-disable*/
```

Comment:
And now I just noticed that [the Markdown-based pages still use this locale.js script](https://github.com/mozilla/testpilot/blob/9e024ee6a7b563263ab20759893fdd3a003ec3ed/frontend/tasks/pages.js#L277).

Reply:
Don't we still need this for locale-based redirects on those pages?